editor/modes/visual-editor/test/inserter.js

```diff
@@ -26,11 +26,17 @@ describe( 'VisualEditorInserter', () => {
 		expect( wrapper.state( 'isShowingControls' ) ).toBe( false );
 	} );
 
-	it( 'should insert paragraph block', () => {
+/*	it( 'should insert paragraph block', () => {
 		const onInsertBlock = jest.fn();
 		const wrapper = shallow(
 			<VisualEditorInserter onInsertBlock={ onInsertBlock } />
```

Comment:
Actually we do not test the "connected" component but just the presentational one, so you just need to provide "mostFrequentlyUsedBlocks" as a prop.